ts/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -156,7 +253,7 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
 
                 // Validate to make sure codeDir doesn't have too many files
                 try {
-                    await validateCodeDir(this.fcTrialConfig.codeDir);
+                    await validateCodeDir(this.fcTrialConfig ? this.fcTrialConfig.codeDir : './');
```

Comment:
If `this.fcTrialConfig.codeDir` is undefined, what does `./` contains?

Reply:
I removed this again. I actually didn't realize, the path is modified within the python logic and felt like setting it to the "current" folder per default (which wouldn't turn out to work as I realized).